app/src/main/java/by/evgen/an11pazniakhomework14/view/MainActivity.kt

```diff
@@ -0,0 +1,286 @@
+package by.evgen.an11pazniakhomework14.view
+
+import android.os.Bundle
+import android.widget.Button
+import android.widget.ImageView
+import android.widget.TextView
+import androidx.appcompat.app.AppCompatActivity
+import androidx.core.content.ContextCompat
+import by.evgen.an11pazniakhomework14.R
+import by.evgen.an11pazniakhomework14.homework15.data.*
+import by.evgen.an11pazniakhomework14.homework15.observer.Editor
+import by.evgen.an11pazniakhomework14.homework15.observer.UpdateDrinksListener
+import by.evgen.an11pazniakhomework14.homework15.util.CatalogClass
+import by.evgen.an11pazniakhomework14.homework15.util.OrderList
+import by.evgen.an11pazniakhomework14.homework15.util.getInfo
+import by.evgen.an11pazniakhomework14.old.CatDataClass
+import by.evgen.an11pazniakhomework14.old.createNewBarsik
+
+class MainActivity : AppCompatActivity() {
+    private lateinit var result: String
+    private lateinit var drinkTextView: TextView
+    private lateinit var catalogTextView: TextView
+    private var iter = 0
+    private var catalogClass: CatalogClass<Spirits> = CatalogClass()
+//    private var isFirstClick = true
+
+    override fun onCreate(savedInstanceState: Bundle?) {
+        super.onCreate(savedInstanceState)
+        setContentView(R.layout.activity_main)
+        findFibCycle(20) //поиск Фибоначчи в цикле
```

Comment:
20 to const

Reply:
Done: moved to the companion object.  Thank you.